app/code/Magento/Customer/Model/Metadata/Form/AbstractData.php

```diff
@@ -64,6 +65,11 @@ abstract class AbstractData
      */
     protected $_localeResolver;
 
+    /**
+     * @var \Magento\Framework\Filesystem\Driver\Http
+     */
+    protected $_http;
```

Comment:
Please make it private and without underscore prefix.
```suggestion
    private $http;
```